### ISSUE TITLE
Service "{{ .Values.tenant.name }}-console" not exist

### ISSUE THREAD

**Issue by Viktor3434** [minio/operator#2254]
I can't access the console.
I noticed that the “service” resource doesn't exist (pointed to by ingress-nginx), and I think that's the problem.

## Expected Behavior

I should get a web ui page minio instead of the 503 error

## Current Behavior

получаю ошибку 503.
при этом API работает (через mc я могу получить доступ т.к. сервис для ingress.api существует)

## Possible Solution

In this case, as a temporary solution, you can of course create a service via extraResources.
but it would be great if the tenant would work out of the box :)

## Steps to Reproduce (for bugs)

1. create a namespace
2. set the operator
3. install a tenant

## Context

log in to the web ui

## Your Environment

* Version used (`minio-operator`):
version: 6.0.0
appVersion: v6.0.0
image.tag: RELEASE.2024-07-13T01-46-15Z
* Environment name and version (e.g. kubernetes v1.17.2): k8s v1.21.5
* Server type and version: managed k8s
* Operating System and version (`uname -a`):
* Link to your deployment file:
my values that differ from the default values

```
secrets:
  name:
  accessKey: 
  secretKey:

tenant:
  name: minio-tenant

  image:
    repository: minio/minio
    tag: RELEASE.2024-07-13T01-46-15Z
    pullPolicy: IfNotPresent

  configuration:
    name: minio-tenant
  configSecret:
    name: minio-tenant
    accessKey:
    secretKey:
    existingSecret: true

  pools:
    - servers: 4
      name: pool-0
      volumesPerServer: 1
      size: 30Gi
      storageClassName: default
  mountPath: /export
  subPath: /data

ingress:
  api:
    enabled: true
    ingressClassName: ""
    labels: { }
    annotations:
      kubernetes.io/ingress.class: nginx
      nginx.ingress.kubernetes.io/proxy-body-size: "0"
      nginx.ingress.kubernetes.io/backend-protocol: https
    tls:
      - hosts:
          - minio-api-test.my.org
        secretName: wild-my-org
    host: minio-api-test.my.org
    path: /
    pathType: Prefix
  console:
    enabled: true
    ingressClassName: ""
    labels: { }
    annotations:
      kubernetes.io/ingress.class: nginx
      nginx.ingress.kubernetes.io/proxy-body-size: "0"
      nginx.ingress.kubernetes.io/backend-protocol: https
    tls:
      - hosts:
          - dp-minio-test.my.org
        secretName: wild-my-org  
    host: dp-minio-test.my.org
    path: /
    pathType: Prefix
```

**Comment by ramondeklein** [comment]
Please note that Operator v6 doesn't have a web UI anymore. See https://github.com/minio/operator/pull/2205.

**Comment by Viktor3434** [comment]
I'm sorry, my mistake.
I meant the tenant.

**Comment by ramondeklein** [comment]
I think this was fixed in v6.0.1 (see https://github.com/minio/operator/pull/2230/files). So please upgrade to v6.0.1 and try again...
